v2/pkg/operators/matchers/compile.go

```diff
@@ -43,6 +43,15 @@ func (m *Matcher) CompileMatchers() error {
 		m.regexCompiled = append(m.regexCompiled, compiled)
 	}
 
+	// Compile and validate binary Values in matcher
+	for _, value := range m.Binary {
+		if decoded, err := hex.DecodeString(value); err != nil {
+			return fmt.Errorf("could not hex decode binary: %s", value)
+		} else {
+			m.binaryCompiled = append(m.binaryCompiled, string(decoded))
```

Comment:
there is no compliation going on here, hence the field should be renamed

Reply:
Done